lib/util/move-file.js

```diff
@@ -25,7 +25,7 @@ function moveFile (src, dest) {
       if (err) {
         if (err.code === 'EEXIST' || err.code === 'EBUSY') {
           // file already exists, so whatever
-        } else if (err.code === 'EPERM' && process.platform === 'win32') {
+        }/* istanbul ignore next */ else if (err.code === 'EPERM' && process.platform === 'win32') {
```

Comment:
This should be `/* istanbul ignore if */` so that we still cover the `else` block below it.

Reply:
that was my first intuition too but then it turns out using `/* istanbul ignore if */` does not really works as I was expecting (**L28** still misses coverage, same as not having the istanbul comment at all)

that's how I ended up moving it to the **L26** and using `/* istanbul ignore else */` instead (the original commit I had in this PR) 😞

Reply:
(posted the bug to istanbuljs here: https://github.com/istanbuljs/istanbuljs/issues/477)